proposal.md

```diff
@@ -7,12 +7,13 @@
 ## Summary
 
 The purpose of the 'RSpec Playground' is to equip students with additional practice before doing TDD with Connect Four. In addition, I believe students do not understand several important concepts, such as: 
-- Writing small and isolated methods.
-- Using a testing pattern, such as 'Arrange Act, and Assert'.
-- Correctly using an explicit subject.
-- How to create different test conditions
-- How to stub methods - either class methods or puts/gets (and that it can be unnecessary to test puts/gets).
-- How to use doubles, verifying doubles, etc.
+- The importance of writing small and isolated methods.
```

Comment:
For this whole list (10-16) I would not use periods, since these are not complete sentences. But that's just my preference. Also I'd probably not capitalize the items on the list. I think both of these potential changes are really your call though.